content/hi/docs/reference/glossary/operator-pattern.md

```diff
@@ -0,0 +1,19 @@
+---
+title: ऑपरेटर पैटर्न(Operator pattern)
+id: operator-pattern
+date: 2019-05-21
+full_link: /docs/concepts/extend-kubernetes/operator/
+short_description: >
+  कस्टम संसाधन का प्रबंधन करने के लिए उपयोग किया जाने वाला एक विशेष नियंत्रक
+
+aka:
+tags:
+- architecture
+---
+[ऑपरेटर पैटर्न](/docs/concepts/extend-kubernetes/operator/) एक सिस्टम डिज़ाइन है जो {{< glossary_tooltip text="नियंत्रक" term_id="controller" >}}
+को एक या अधिक कस्टम संसाधनों से जोड़ता है।
+
+<!--more-->
+आप कुबेरनेट्स के हिस्से के रूप में आने वाले अंतर्निहित नियंत्रकों से परे, अपने क्लस्टर में नियंत्रकों को जोड़कर कुबेरनेट्स का विस्तार कर सकते हैं।
+
+यदि कोई चल रहा एप्लिकेशन नियंत्रक के रूप में कार्य करता है और उसके पास नियंत्रण विमान में परिभाषित कस्टम संसाधन के विरुद्ध कार्य करने के लिए एपीआई पहुंच है, तो यह ऑपरेटर पैटर्न का एक उदाहरण है।
```

Comment:
@Omkar0114  translation of this line is not correct completely. Rewrite this line

Reply:
Which line are you saying exactly? @kundan2707

Reply:
@Omkar0114 यदि कोई चल रहा एप्लिकेशन नियंत्रक के रूप में कार्य करता है और उसके पास नियंत्रण विमान में परिभाषित कस्टम संसाधन के विरुद्ध कार्य करने के लिए एपीआई पहुंच है, तो यह ऑपरेटर पैटर्न का एक उदाहरण है।

Reply:
Done with this line @kundan2707 ✅